src/components/AnnotationBrowser/AnnotationList.vue

```diff
@@ -301,20 +302,58 @@ export default class AnnotationList extends Vue {
     return this.tableOptions.itemsPerPage;
   }
 
+  get vDataTable() {
+    const vDataTableParent = this.$refs.dataTable as any;
+    if (!vDataTableParent) {
+      return null;
+    }
+    const vDataTable = vDataTableParent?.$children?.[0];
+    if (!vDataTable) {
+      return null;
+    }
+    return vDataTable;
```

Comment:
Can it be undefined?
Otherwise, I think you can have only one return statement

Reply:
It can be null or a vDataTable element, I changed it to a one liner